ush/global_ens/evs_global_ens_atmos_grid2grid.sh

```diff
@@ -461,10 +514,22 @@ if [ $verify = precip ] ; then
             fi      
           fi
         fi
-        chmod +x run_${modnam}_ccpa${apcp}_valid_at_t${vhour}z_${metplus_job}.sh
+
+        # Indicate sub-task is completed for restart 
+	echo ">$COMOUTsmall_precip/run_${modnam}_ccpa${apcp}_valid_at_t${vhour}z_${metplus_job}.completed" >> run_${modnam}_ccpa${apcp}_valid_at_t${vhour}z_${metplus_job}.sh
+
+	# Save files for restart
+	echo "mkdir -p $COMOUTsmall_precip/restart/grid2grid/run_${modnam}_ccpa${apcp}_valid_at_t${vhour}z/stat" >> run_${modnam}_ccpa${apcp}_valid_at_t${vhour}z_${metplus_job}.sh
+
+	echo "cp -rfu $WORK/grid2grid/run_${modnam}_ccpa${apcp}_valid_at_t${vhour}z/stat/${modnam} $COMOUTsmall_precip/restart/grid2grid/run_${modnam}_ccpa${apcp}_valid_at_t${vhour}z/stat" >> run_${modnam}_ccpa${apcp}_valid_at_t${vhour}z_${metplus_job}.sh
```

Comment:
Can we add two checks here?

SENDCOM = YES
The files it is trying to copy exist?

Reply:
Please see the answers in the previous section.